index.d.ts

```diff
@@ -31,75 +31,75 @@ declare module 'fluture' {
   }
 
   export function after<L, R>(duration: number, value: R): Future<L, R>
```

Comment:
You can add some documentation here

```ts
  /** Creates a Future which resolves with the given value after n milliseconds. */
  export function after<L, R>(duration: number, value: R): Future<L, R>
  export function after<L, R>(duration: number): (value: R) => Future<L, R>
```

will show up in the IDE (for example vscode)